serverlessworkflow/modules/ROOT/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -92,8 +125,7 @@ sonataflowplatforms.sonataflow.org   2023-03-08T18:31:15Z
 sonataflows.sonataflow.org   2023-03-08T18:31:15Z
```

Comment:
This section on operator CRDs, is this necessary?  The wording is a bit confusing to me ("you must first install" gives the impression the user still needs to do something), and by this time the operator should have done that already?

Reply:
Yes, the YAML applied before should've applied the CRDs. Let me rephrase this section.